tests/core/test_views_version.py

```diff
@@ -9,34 +9,32 @@
 
 
 class VersionViewTest(BaseWebTest, unittest.TestCase):
-    def tearDown(self):
-        try:
-            os.remove('version.json')  # Clean-up file at default location.
-        except OSError:
-            pass
+    def setUp(self):
```

Comment:
But we also need it in the `tearDown` don't we? So the memoization doesn't stay around for following tests? Or are you going to call `self.addCleanup` or something?